proto/dclupgrade/query.proto

```diff
@@ -21,6 +22,16 @@ service Query {
 		option (google.api.http).get = "/dcl/dclupgrade/proposed-upgrades";
 	}
 
+// Queries a ApprovedUpgrade by index.
+	rpc ApprovedUpgrade(QueryGetApprovedUpgradeRequest) returns (QueryGetApprovedUpgradeResponse) {
```

Comment:
Looks like openapi should also be updated

Reply:
updated